crates/web5/src/lib.rs

```diff
@@ -4,8 +4,7 @@ pub mod dids;
 
 mod datetime;
 pub mod errors;
-mod http;
-mod jose;
+pub mod jose;
```

Comment:
didn't catch this before merge, but why are we exposing `jose` now? IIRC we wanted to keep this as an implementation detail.

Reply:
tbdex uses detached compact jws as it's signature field